docs/common/yum.md

```diff
@@ -74,6 +74,20 @@ You must install and enable those first.
     The OSG repositories must have a better (numerically lower) priority than the EPEL repositories;
     you might have dependency resolution ("depsolving") issues otherwise.
 
+### Automatic Updates
+
+We strongly recommend against automatic updates for production services.
+You want to only change software versions during a controlled downtime (or, at least, while a human is watching);
```

Comment:
I try to stay way from "you want"/"you should"/"you should". Let's maybe reword this to:

> For production services, we suggest only changing software versions during controlled downtime. Therefore we recommend security-only automatic updates or disabling automatic updates entirely. 

Then give instructions for both cases.